src/SFA.DAS.AssessorService.Application.Api.UnitTests/Controllers/Register/Query/GetStandardVersionsByOrganisationIdAndStandardReferenceTests.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using FluentAssertions;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using NUnit.Framework.Internal;
+using SFA.DAS.AssessorService.Api.Types.Models;
+using SFA.DAS.AssessorService.Api.Types.Models.AO;
+using SFA.DAS.AssessorService.Application.Api.Controllers;
+using SFA.DAS.AssessorService.Domain.Consts;
+
+namespace SFA.DAS.AssessorService.Application.Api.UnitTests.Controllers.Register.Query
```

Comment:
Just need to change the unit test class and file name to match new GetAppliedStandardVersionsForEPAO

Reply:
Fixed

Reply:
File still needs renaming GetAppliedStandardVersionsForEpaoTests

Reply:
Fixed and commited